keylime/web_util.py

```diff
@@ -252,17 +253,18 @@ def get_tls_options(component, is_client=False, logger=None):
     # Peer certificate verification is enabled by default
     verify_peer_certificate = True
 
-    trusted_ca = config.get(component, ca_option)
-    if not trusted_ca:
+    trusted_ca: List[str]
```

Comment:
What does this exactly do in python? Because in my understanding this does not create a variable with the name trusted_ca.

Reply:
I dropped this line now.